CONTRIBUTING.md

```diff
@@ -69,16 +69,23 @@ We provide a script that can automatically create all the necessary files for yo
   - Use existing Sass variables for spacing (margin, padding, sizing, etc).
   - Use [`themed`](https://github.com/iTwin/iTwinUI/blob/main/src/style/theme.scss#L430) mixin wherever color is involved.
 - Define classes for your mixin in `src/[component-name]/classes.scss` file.
-  - *Running the `createComponent` command will do this for you.*
+  - _Running the `createComponent` command will do this for you._
 - Make sure your component index and classes are imported in `src/index.scss` and `src/classes.scss`.
-  - *Running the `createComponent` command will also do this for you but you need to manually sort the imports alphabetically.*
+  - _Running the `createComponent` command will also do this for you but you need to manually sort the imports alphabetically._
 - Write tests for your new component in `backstop/tests/[component-name].html` and `backstop/scenarios/[component-name].js`. See [Testing](#Testing) section below.
 - After running `yarn build` you can open minified html in browser to check up, how your component looks like from `backstop/minified`.
 
 ### Testing
 
 For running tests you will need [Docker](https://www.docker.com/products/docker-desktop). It helps to avoid cross-platform rendering differences.
 
+#### How to set up Docker:
+
+- Download and install [Docker](https://www.docker.com/products/docker-desktop) and make sure Docker is running
```

Comment:
Do we need to specify this again? The line right above says you will need docker, so maybe we can just say "Make sure Docker is running".

Reply:
Ohh, right, that makes sense. Edited it to just `Make sure Docker is running`